test/Ocelot.AcceptanceTests/RoutingTests.cs

```diff
@@ -1,5 +1,5 @@
-using Microsoft.AspNetCore.Http;
-using Ocelot.Configuration.File;
+using Microsoft.AspNetCore.Http;
+using Ocelot.Configuration.File;

```

Comment:
Zero diff!
Line endings?
What IDE do you use to write C# code?
Please try to remove wrong line endings. So, don't change them!

Reply:
Using Vscode